tsuru/installer/dm/dockermachine.go

```diff
@@ -200,10 +200,6 @@ func (d *DockerMachine) createRegistryCertificate(hosts ...string) error {
 	return generator.GenerateCert(certOpts)
 }
 
-func (d *DockerMachine) DeleteAll() error {
```

Comment:
@cezarsa I didn't understand why we removed that.

Reply:
It wasn't called anywhere. I found it running `staticcheck -unused.whole-program` and filtering the false positives due to test code.